objectbox/lib/src/native/sync.dart

```diff
@@ -554,7 +554,7 @@ class Sync {
   /// Create a Sync annotation, enabling synchronization for an entity.
   const Sync();
 
-  static late final bool _syncAvailable = C.sync_available();
+  static late final bool _syncAvailable = C.has_feature(OBXFeature.Sync);
```

Comment:
I guess this just snuck in here? Fine to keep, just double checking if not by accident.

Reply:
yeah, noticed the use of the deprecated function so just changed it in the meantime